test/v200/utils/api/test_utils.go

```diff
@@ -9,7 +9,7 @@ import (
 	"strings"
 	"testing"
 
-	commonUtils "github.com/devfile/api/v2/test/v200/utils/common"
+	commonUtils "github.com/mmulholla/api/test/v200/utils/common"
```

Comment:
This needs cleanup before merge if nothing else.

Reply:
oops, was using the same branch for testing. Fixed now